asset_dashboard/templates/asset_dashboard/partials/forms/add_edit_phase_form.html

```diff
@@ -11,14 +11,27 @@
 <div class="row">
   <div class="border border-secondary rounded shadow-sm col m-2 p-3 card">
 
-    <form method="post">
+    <form method="post" id="phaseForm" name="phaseForm">
       {% csrf_token %}
 
       {{ form.errors }}
-  
+
       <div class="row col justify-content-between">
-        <h3>{% if phase %}Update Phase{% else %}Create Phase{% endif %}</h3>
-        <input type="submit" value="Save Phase" class="btn btn-info">
+        <h3>
+          {% if phase %}
+            Update Phase
+            <div class="icon-loader phase-icon"></div>
+            <div class="icon-saved phase-icon">
+              <i class="fas fa-check-square" title="Saved" aria-label="Saved"></i>
+            </div>
+            <div class="icon-failed phase-icon">
+              <i class="fas fa-exclamation-circle" title="Save Failed" aria-label="Save Failed"></i>
+            </div>
+          {% else %}
+            Create Phase
+          {% endif %}
+        </h3>
+        <button id="save-phase" class="btn btn-info">Save Phase</button>
```

Comment:
I kept all of the existing save buttons for better UX. It's nice to have a manual save option for sanity's sake.